lrtable/src/lib/mod.rs

```diff
@@ -69,7 +74,7 @@ impl From<u32> for StIdx {
 
 impl From<usize> for StIdx {
     fn from(v: usize) -> Self {
-        if v > u16::max_value() as usize {
+        if v > usize::from(u16::max_value()) {
```

Comment:
Just below this (but not in the diff) there are a couple more `as u16` that could be `u16::from`

Reply:
I've only done the ones that Clippy complained about. My plan is to do a much bigger tidy up of the integer conversions in a later PR (which is now, hopefully, much easier than it was before).

Reply:
Clippy complains about the ones I mentioned, but I think it's reasonable to defer that to a separate PR, since you haven't touched those lines here.